index.js

```diff
@@ -47,6 +48,51 @@ if (args.sendgrid_cc && args.sendgrid_cc !== '') {
 	process.env.SENDGRID_CC = args.sendgrid_cc;
 }
 
+//A common location for performing DB upgrades
+async function updateDB() {
+	await sqlite.open('./gg.db');
+	let version = -1;
+	var r = null;
```

Comment:
Super nitpicky (and my fault for not putting this in eslint rules) but prefer let and const instead of var. It totally doesn't matter though (besides being consistent), so feel free to ignore it.